providers/class-two-factor-fido-u2f.php

```diff
@@ -73,21 +73,10 @@ protected function __construct() {
 		require_once TWO_FACTOR_DIR . 'providers/class-two-factor-fido-u2f-admin.php';
 		Two_Factor_FIDO_U2F_Admin::add_hooks();
 
-		wp_register_script(
-			'fido-u2f-api',
-			plugins_url( 'includes/Google/u2f-api.js', dirname( __FILE__ ) ),
-			null,
-			self::asset_version(),
-			true
-		);
-
-		wp_register_script(
-			'fido-u2f-login',
-			plugins_url( 'js/fido-u2f-login.js', __FILE__ ),
-			array( 'jquery', 'fido-u2f-api' ),
-			self::asset_version(),
-			true
-		);
+		// Ensure the script dependencies have been registered before they're enqueued at a later priority.
+		add_action( 'admin_enqueue_scripts', array( __CLASS__, 'enqueue_scripts' ), 5 );
```

Comment:
Use action priority 5 to ensure the scripts have been registered before they're enqueued later at priority 10.